ios/browser/api/ads/BUILD.gn

```diff
@@ -48,6 +48,7 @@ ios_objc_mojom_wrappers("ads_mojom_wrappers") {
   mojom_target =
       "//brave/vendor/bat-native-ads/include/bat/ads/public/interfaces"
   sources = [
+    "//brave/components/brave_federated/public/interfaces/brave_federated.mojom",
```

Comment:
This can be removed as I see no public exposure of the brave_federated types in `brave_ads.h`.  If we do need to expose it then additional changes are needed to include it in the iOS framework's public headers